src/main/scala/com/databricks/labs/tika/TikaExtractor.scala

```diff
@@ -8,6 +8,7 @@ import org.apache.tika.parser.ocr.TesseractOCRConfig
 import org.apache.tika.parser.pdf.PDFParserConfig
 import org.apache.tika.parser.{AutoDetectParser, ParseContext, Parser}
 import org.apache.tika.sax.BodyContentHandler
+import org.apache.poi.util.IOUtils
```

Comment:
Can you explicitly add this dependency to pom.xml? Seems that we rely on this to be implicitly pulled in

Reply:
It's a sub-dependency of tika for MS Office file text extraction. I'll see which exact version they are using.

Reply:
The latest version `5.2.5` of POI seems to pass our tests. Will add explicitly.